edx_lint/tamper_evident.py

```diff
@@ -26,21 +26,21 @@ def write(self, text, hashline=b"# {}"):
         be changed to accommodate different file syntaxes.
 
         Arguments:
-            text (byte string): the contents of the file to write.
+            text (text string): the contents of the file to write.
```

Comment:
Ok. I tend to refer to the distinction as text vs bytes, and have seen others do the same, but I'm fine w/ unicode as well.